packages/flutter_tools/lib/src/ios/code_signing.dart

```diff
@@ -116,6 +118,53 @@ Future<Map<String, String>?> getCodeSigningIdentityDevelopmentTeam({
     return null;
   }
 
+  final String? developmentTeam = await _getCodeSigningIdentityDevelopmentTeam(
+    processManager: processManager,
+    platform: platform,
+    logger: logger,
+    config: config,
+    terminal: terminal,
+    shouldExitOnNoCerts: true,
+  );
+
+  if (developmentTeam == null) {
+    return null;
+  }
+
+  return <String, String>{
+    'DEVELOPMENT_TEAM': developmentTeam,
+  };
+}
+
+Future<String?> getCodeSigningIdentityDevelopmentTeam({
```

Comment:
Do we need this wrapper, or can we make `_getCodeSigningIdentityDevelopmentTeam` public?

Reply:
I was trying to hide the implementation detail of `shouldExitOnNoCerts` from the public APIs it's true for `getCodeSigningIdentityDevelopmentTeamBuildSetting` and false for `getCodeSigningIdentityDevelopmentTeam`.

Reply:
makes sense